### PR TITLE
(minikube) update to new github style

### PR DESCRIPTION
**Pull request by tunisiano187**
## Description
update to new github style

## Motivation and Context
Github changes break the update
fixes https://github.com/chocolatey-community/chocolatey-packages/issues/1994

## How Has this Been Tested?
Powershell locally

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [ ] I have updated the package description and it is less than 4000 characters.
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

**Comment by pauby** [comment]
Can you see #2007 and #2013 recently submitted to resolve this GitHub API issue. We should use one way to do this in the repo.

**Comment by tunisiano187** [comment]
OK, so I need to wait until that PR is merged.

**Review by AdmiringWorm — CHANGES_REQUESTED** [review]
Before we merge this PR in, can you update the commit messages with the same style already being used in the repository.

That is all commit messages should be prefixed with the package name it is for.
So in your case, the commit message should start with `(minikube)`, also rebase the PR instead of doing a merge commit into the PR (ie, get rid of the merge commit).

**Pull request closed without merging**
